sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/ExperimentStore.java

```diff
@@ -35,25 +12,22 @@
  * @author Rob Davey
  * @since 0.0.2
  */
-public interface ExperimentStore extends Store<Experiment> {
+public interface ExperimentStore extends SaveDao<Experiment> {
 
   /**
    * List all Experiments that are part of a Study given a Study ID
    * 
-   * @param studyId
-   *          of type long
+   * @param studyId of type long
    * @return Collection<Experiment>
-   * @throws IOException
-   *           when
+   * @throws IOException when the objects cannot be retrieved
    */
   public Collection<Experiment> listByStudyId(long studyId) throws IOException;
 
   /**
    * List all persisted objects
    * 
    * @return Collection<Experiment>
-   * @throws IOException
-   *           when the objects cannot be retrieved
+   * @throws IOException when the objects cannot be retrieved
    */
   public Collection<Experiment> listAllWithLimit(long limit) throws IOException;
```

Comment:
`listByStudyId` is unused and can be removed. `listAllWithLimit` is used in the service, but the service method is unused, so it can be removed from both